zabbix_auto_config/utils.py

```diff
@@ -32,7 +32,8 @@ def validate_host(host):
         "properties",
         "proxy_pattern",
         "siteadmins",
-        "sources"
+        "sources",
+        "tags"
```

Comment:
When using git a trailing comma will lead to smaller diffs. Add this to last item?